aicsimageio/tests/writers/test_ome_tiled_tiff_writer.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import urllib
+from typing import Callable, Tuple, Union
+
+import numpy as np
+import pytest
+from ome_types import OME, to_xml
+
+from aicsimageio.readers import OmeTiffReader
+from aicsimageio.writers import OmeTiffWriter
+from aicsimageio.writers.bfio_writers import OmeTiledTiffWriter
+
+from ..conftest import LOCAL, array_constructor, get_resource_write_full_path
+
+
+@array_constructor
+@pytest.mark.parametrize(
+    "write_shape, expected_read_shape, expected_read_dim_order",
+    [
+        ((5, 16, 16), (1, 1, 5, 16, 16), "TCZYX"),
+        ((5, 16, 16), (1, 1, 5, 16, 16), "TCZYX"),
+        # OmeTiffReader is curently reordering dims to TCZYX always
+        ((5, 1, 16, 16), (1, 5, 1, 16, 16), "TCZYX"),
+        ((5, 10, 16, 16), (1, 5, 10, 16, 16), "TCZYX"),
+        ((5, 10, 16, 16), (1, 5, 10, 16, 16), "TCZYX"),
+        ((16, 16), (1, 1, 1, 16, 16), "TCZYX"),
+        ((1, 2, 3, 4, 5), (1, 2, 3, 4, 5), "TCZYX"),
+        ((2, 3, 4, 5, 6), (2, 3, 4, 5, 6), "TCZYX"),
+        ((2, 3, 4, 5, 6), (2, 3, 4, 5, 6), "TCZYX"),
+    ],
+)
+@pytest.mark.parametrize("filename", ["e.ome.tiff"])
```

Comment:
are we using "e.ome.tiff" anywhere else? I think I originally chose those stupid filenames because I was writing code quickly but can you make this have a better filename to link to this test? I.e. "tiled.ome.tiff"